src/pickup.cpp

```diff
@@ -86,7 +86,8 @@ interact_results interact_with_vehicle( vehicle *veh, const tripoint &pos,
     const bool remotely_controlled = g->remoteveh() == veh;
     const int washing_machine_part = veh->part_with_feature( veh_root_part, "WASHING_MACHINE" );
     const bool has_washmachine = washing_machine_part >= 0;
-    bool washing_machine_on = ( washing_machine_part == -1 ) ? false : veh->parts[washing_machine_part].enabled;
+    bool washing_machine_on = ( washing_machine_part == -1 ) ? false :
```

Comment:
Brackets ain't needed here, right?

Reply:
It's a god idea, the following ? is low precedence, so it can easily surprise you.